osu.Game/Overlays/WikiOverlay.cs

```diff
@@ -148,6 +157,7 @@ private void onSuccess(APIWikiPage response)
 
         private void onFail()
         {
+            path.Value = "error";
```

Comment:
`path.Value` is being used in the very next line, so this partially breaks the error display.

I'll get this (should be enough to reorder the lines, in theory).

Reply:
Actually it is not that easy because in the test there's a situation where *two* requests to the same page are in flight, and the first one sets the path to `error`, so even if I reorder the write on the first one, then the second will read that value and still break the error display...

Was there any particular reason to mess with the current path? Could a separate boolean flag be used for this instead or something?

Reply:
It was an afterthought to fix the linked discussion. Obviously not tested properly outside of the test scene and needs further thought..

Reply:
I've pushed a fix for this (https://github.com/ppy/osu/pull/21772/commits/9a2cc043611ab6e78feef7c18958747baa795061). Please check if you're OK with it.

Reply:
seems okay 👍